api/src/setupPlugins.ts

```diff
@@ -0,0 +1,14 @@
+import { setup } from 'powerhouse-switchboard-plugin-demo';
+import { getUserCrud } from './modules/User'
+import { getSessionCrud } from './modules/Session'
+import {API_ORIGIN} from './env'
+
+export const setupAll = () => {
+  setup({
+    apiOrigin: API_ORIGIN,
+    crud: {
+      getUserCrud,
+      getSessionCrud
+    }
```

Comment:
Can't you have this crud object already "prebuilt" if you will provide `prisma` to the setup?

Reply:
it should be possible, although requires some alterations in the original concept where `getCrud` functions are provided to the `getChallengeCrud` function. Since the challenge related functionality will be removed from the pr - this discussion point seems less relevant as of now. But generally i would just orient the direction of the decision so that the types would be easier to handle.